docs/Interoperability_tests_guide/Using_API_Jenkins_integration.md

```diff
@@ -0,0 +1,312 @@
+# Jenkins/GitLab integration with interop tests API
+
+If you would like to automate your interoperability tests, you can use the Coiote DM API and integrate it with a CI/CD environment like Jenkins or GitLab.
+Follow the guide below to learn how to configure the integration, run tests and summarize your test execution using these tools.
+
+!!! note
+    The following instruction is based on integration with Jenkins. To integrate with GitLab, you can follow the same steps, but with slight adjustments - for details, please see subsection on [GitLab](#gitlab-configure-and-run-pipeline).    
+
+## Prerequisites
+
+- An active Jenkins and GitLab account.
+- A GitLab project repository.
```

Comment:
It also can be github repository (url to repository has to be typed in configuration of Jenkins Pipeline, or gitlab CI/CD must be available and configured for this repository url)

Reply:
I guess it could be any git repository. Jenkins supports all git repositories gitlab, github, bitbucket etc.

Reply:
ok, changed to "A Git project repository"